### ISSUE TITLE
error session in php 7.2

### ISSUE THREAD

**Issue by baihaqyaviq** [dappur/framework#22]
in php 7.2 error warning:
run on windows 7 php 7.2
```
Warning: session_set_cookie_params(): Cannot change session cookie parameters when session is active in D:\WWW_PROJECT\PROJECT\framework\public\index.php on line 3

and 

```
Fatal error: Uncaught RuntimeException: Unexpected data in output buffer. Maybe you have characters before an opening <?php tag? in D:\WWW_PROJECT\PROJECT\framework\vendor\slim\slim\Slim\App.php:604 Stack trace: #0 D:\WWW_PROJECT\PROJECT\framework\vendor\slim\slim\Slim\App.php(316): Slim\App->finalize(Object(Slim\Http\Response)) #1 D:\WWW_PROJECT\PROJECT\framework\public\index.php(30): Slim\App->run() #2 {main} thrown in D:\WWW_PROJECT\PROJECT\framework\vendor\slim\slim\Slim\App.php on line 604

**Comment by jeffmthomas** [comment]
Are you encountering this on a fresh clone and migration?

**Comment by baihaqyaviq** [comment]
Yes, in fresh clone,

**Comment by jeffmthomas** [comment]
According to [this answer](https://stackoverflow.com/a/47731428), `session_set_cookie_params()` needs to be called before the session is started.  I am sure that I read somewhere a while back the exact opposite.  Howevert, if you switch lines 2 and 3 it should correct your issue.  This is fixed in the blog branch and will be merged in with the next release.

**Comment by jeffmthomas** [comment]
Applied the fix from the stack post in v2.1. Please let me know if you have any other issues with this.
